eval/src/tests/eval/interpreted_function/interpreted_function_test.cpp

```diff
@@ -53,9 +52,8 @@ struct MyEvalTest : test::EvalSpec::EvalTest {
         if (is_supported && !has_issues) {
             vespalib::string desc = as_string(param_names, param_values, expression);
             SimpleParams params(param_values);
-            verify_result(SimpleTensorEngine::ref(), *function, false,    "[untyped simple] "+desc, params, expected_result);
-            verify_result(DefaultTensorEngine::ref(), *function, false,   "[untyped prod]   "+desc, params, expected_result);
-            verify_result(DefaultTensorEngine::ref(), *function, true,    "[typed prod]     "+desc, params, expected_result);
+            verify_result(SimpleTensorEngine::ref(),      *function, false, "[untyped simple] "+desc, params, expected_result);
```

Comment:
I suggest dropping typed/untyped concept (always typed) and replacing SimpleTensorEngine with SimpleValue here.

Reply:
done.